tagstore/backends/tagstore_django/templatetags/tagstore_admin.py

```diff
@@ -2,21 +2,13 @@
 from django import template
 
 from tagstore.backends.django import DjangoTagstore
-from tagstore.backends.tagstore_django.models import Taxonomy
 
 register = template.Library()
 
 
 @register.inclusion_tag('admin/tagstore_django/tag_hierarchy.html')
-def tag_hierarchy():
-    """ Renders a hierarchical view in HTML of all Tag objects. """
+def tag_hierarchy(taxonomy_uid):
+    """ Renders a hierarchical view in HTML of Tag objects. """
     tagstore = DjangoTagstore()
```

Comment:
@SSPJ I figured out the issue: `taxonomy_uid` is coming in as a string here. If you add `taxonomy_uid = int(taxonomy_uid)` to this method before calling `get_tags_in_taxonomy_hierarchically_as_dict`, you won't get the `KeyError` anymore.